src/assets/images/LeafFillIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import { createSvgIcon } from "@mui/material";
+
+const LeafFillIcon = createSvgIcon(
```

Comment:
Couldn't find MUI icons that would fit the Figma design but looked into MUI icons a little more. I found this nice conversion/wrapping option that we should probably follow through and refactor all our previous custom icons to maintain consistency. 

https://mui.com/material-ui/icons/#createsvgicon

For our Figma svgs it must be noted that we need to set the `width` and `height` to `24px` to allow for the sizing to match up with MUI icons but otherwise I don't see any other issues or further customization needed. They act exactly like MUI icons and `sx` is available to change style, font-size and color of icons as needed.